crates/bevy_tasks/src/slice.rs

```diff
@@ -32,14 +35,14 @@ pub trait ParallelSlice<T: Sync>: AsRef<[T]> {
     /// - [`ParallelSlice::par_splat_map`] for mapping when a specific chunk size is unknown.
     fn par_chunk_map<F, R>(&self, task_pool: &TaskPool, chunk_size: usize, f: F) -> Vec<R>
     where
-        F: Fn(&[T]) -> R + Send + Sync,
+        F: Fn(&[T], usize) -> R + Send + Sync,
```

Comment:
A bit of a nit, but I always found index ,value to be more intuitive with how often key-value pairs show up in that order.

Reply:
Good point, also mirrors Rust's `.enumerate()`